tool/jt.rb

```diff
@@ -612,8 +612,6 @@ def run_ruby(*args)
       if ENV["RUBY_BIN"]
         # Assume Graal is available if RUBY_BIN is set or that we are running
         # a non-TruffleRuby, such as MRI.
-      else
-        vm_args << '-Xgraal.warn_unless=false'
       end
     end
```

Comment:
This entire (outer) `else` branch can be removed.
`# This will also warn if it's not.` also needs to be adapted/removed as it's no longer true.